### ISSUE TITLE
cryptocoin@guantanamoe : Applet doesn't launch on cinnamon (re)start

### ISSUE THREAD

**Issue by enguerran-popup** [linuxmint/cinnamon-spices-applets#1765]
* Applet version/Build date : **1.4 / 2018-03-27, 15:33 UTC**
 * Cinnamon version : **Cinnamon 3.6.7**
 * Distribution : **Linux Mint 18.3 Sylvia** 
 * Graphics hardware *and* driver used : **Nvidia Geforce GTX 1050 - driver 390.48**
 * Snippet or attachment of ~/.cinnamon/glass.log : **[glass.log](https://github.com/linuxmint/cinnamon-spices-applets/files/1899019/glass.log)**
 * Snippet or attachment of ~/.xsession-errors : **[.xsession-errors](https://github.com/linuxmint/cinnamon-spices-applets/files/1899036/xsession-errors.log)**

**Notify author of applet**
@imDeprecated 

**Issue**

Applet works fine after being added but it won't launch on cinnamon (re)start.

```
error t=2018-04-11T15:25:36Z this.settingsData[key].value is null
error t=2018-04-11T15:25:36Z [Applet "cryptocoin@guantanamoe"]: Failed to evaluate 'main' function on applet: cryptocoin@guantanamoe/57
error t=2018-04-11T15:25:36Z Could not load applet cryptocoin@guantanamoe
```

**Comment by icarter09** [comment]
Cinnamon 3.6.7
Linux Mint 18.3
Applet version 2018-05-13 08:33 UTC

@enguerran-popup I just attempted to verify this issue. I was able to reproduce the problem the first time I added the applet and then restarted Cinnamon (Applet did not reload). I added the applet back and waited for it load. Now that the applet is loaded, I have been able to restart Cinnamon several times and the applet is loaded correctly on each restart. 

Hopefully pulling down the latest version of the applet will fix your problem. Thanks for creating this ticket, else I wouldn't have found out about this applet. It's pretty nice.
